frontend/src/types/Theme.ts

```diff
@@ -1,7 +1,7 @@
 export interface Header {
     nextExperimentButtonText: string;
     aboutButtonText: string;
-    showScore: boolean;
+    showScore: boolean;    
```

Comment:
This is not consistent with what's implemented in the backend now (i.e., `score` dictionary).